workflow.go

```diff
@@ -118,8 +117,7 @@ func CartWorkflow(ctx workflow.Context, state CartState) error {
 			selector.AddFuture(workflow.NewTimer(ctx, abandonedCartTimeout), func(f workflow.Future) {
 				sentAbandonedCartEmail = true
 				ao := workflow.ActivityOptions{
-					ScheduleToStartTimeout: time.Minute,
-					StartToCloseTimeout:    time.Minute,
+					ScheduleToCloseTimeout: time.Minute,
```

Comment:
Its a pretty low value for schedule to close. Do you really want activities to completely fail if they haven't been succesful only after 1 minute? It would mean at most 4-5 retries can happen ever. I would define `StartToClose` and not `ScheduleToClose` as `ScheduleToClose` is rarely needed.